spec/system/dashboard_system_spec.rb

```diff
@@ -472,6 +480,56 @@ def create_next_product_drive_donation(donation_date:)
           end
         end
       end
+
+      describe "Product drive behaviour with Mixed Donation types" do
+        before do
+          @item_quantity = item_quantities.to_enum
+          @money_raised_on_each_product_drive = 123 # This is arbitrary, but needs to be different than other donations
+          @money_raised_on_each_manufacturer_donation = 231 # This is arbitrary, but needs to be different than other donations
+
+          # Create some number of Product Drives
+          # Keep local copy of names so examples can create expected values
+          # without relying on fetching info from production code
+          @product_drives = (1..rand(2..5)).map do
+            name = "Product Drive #{_1}"
+            OpenStruct.new name: name, drive: create(:product_drive, name: name)
+          end
+
+          @product_drive_donations = 2.times.map do
+            create_next_product_drive_donation donation_date: test_time
+          end
+
+          # create a different donation -- which shouldn't get included in to our totals or shown under product drive
+          quantity_in_donation = @item_quantity.next
+          manufacturer = create :manufacturer, name: "Manufacturer for product drive test", organization: @organization
+          create :manufacturer_donation, :with_items, manufacturer: manufacturer, issued_at: test_time, item_quantity: quantity_in_donation, storage_location: storage_location, organization: @organization, money_raised: @money_raised_on_each_manufacturer_donation
+
+          org_dashboard_page.visit
+        end
+
+        it "only counts product drive donations for product drive" do
+          expect(org_dashboard_page.product_drive_total_donations).to eq @product_drive_donations.map(&:quantity).sum
+        end
+        it "only counts product drive money raised" do
+          expect(org_dashboard_page.product_drive_total_money_raised).to eq @product_drive_donations.map(&:money_raised).sum
+        end
+        it "only shows product drive donations as product drive donations" do
```

Comment:
```suggestion
        it "only counts product drive donations for product drive" do
          expect(org_dashboard_page.product_drive_total_donations).to eq @product_drive_donations.map(&:quantity).sum
        end
        
        it "only counts product drive money raised" do
          expect(org_dashboard_page.product_drive_total_money_raised).to eq @product_drive_donations.map(&:money_raised).sum
        end
        
        it "only shows product drive donations as product drive donations" do
```
Spacing to make this personally more readable for me.